### PR TITLE
Fix username/role cutoff in user cards

### PR DESCRIPTION
**Pull request by rzumer**
* Set 400px as the minimum width instead of fixed width
* Add a margin between the username and user role

Higher fixed width would work too but since roles are variable-length it seems saner to just give it a variable width, in practice it can't be longer than ~500px due to the username length cap and current role lengths.

Before
image
After
image

Tested only in the inspector, not a local environment because WSL v1 doesn't support Docker, Sail couldn't run and I gave up when populating the database failed probably because of some missing init step.

### CODE REVIEW

File: resources/css/card.css
Comment (wescopeland) [inline resources/css/card.css:13]: Please use a Tailwind class such as `ml-4` instead of adding a new class name.

**Review by wescopeland — CHANGES_REQUESTED** [review]

File: resources/css/card.css
Comment (rzumer) [reply resources/css/card.css]: OK, done

**Review by wescopeland — APPROVED** [review]

**Comment by Alexdatadestroyer** [comment]
So, is this already a solution for #1625 that I reported one day earlier the date of this Pull Request? Amazing!

However, User Motto will give some issues when link is posted on it. I gave more details on my issue earlier that day...

**Comment by wescopeland** [comment]
Hi @Alexdatadestroyer,

It is unlikely this change will be accepted into the project. Due to org permissions, I do not have the ability to dismiss my approval, but it's safe to disregard.

https://github.com/RetroAchievements/RAWeb/pull/1429 is an active PR containing a significant rework of site tooltip functionality. Once it is merged, I will have a subsequent PR that fixes numerous tooltip issues (including https://github.com/RetroAchievements/RAWeb/issues/1625)

**Pull request closed without merging**
